sonar-batch-protocol/src/main/gen-java/org/sonar/server/source/db/FileSourceDb.java

```diff
@@ -2018,6 +2047,7 @@ public Builder clearOverallConditions() {
         return this;
       }
 
+      // optional int32 overall_covered_conditions = 14;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)